### PR TITLE
Feat/59 update text on about page

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
Adds some additional text and links to the about page footer

Testing:
- go to about page. check it matches text in the document https://wcmc.sharepoint.com/:w:/s/ActionAgendaPhaseThree/EQcTg9YApGhBkv5i78Sgj7sB7_TOt5I2ymW21rKKuKy5YQ?e=0gxIdl

### CODE REVIEW

File: config/locales/about/en.yml [hunk 1]
```diff
@@ -11,15 +11,23 @@ en:
...
-              and inspire each other. Commitments for Nature will provide an opportunity to facilitate 
-              partnerships and collaboration, as well as test innovative approaches to conservation."
+            continue to raise public awareness and inspire others to safeguard, restore and invest in nature.
+            The Commitments for Nature platform will complement and support the 
+            <a href='https://www.cbd.int/action-agenda/contribute/' target='blank'>Action Agenda</a> 
+            by working with it in tandem, and displaying the progress being made towards different commitments.
+            Another important international body that the Commitments for Nature Platform links to is the 
+            <a href='https://citieswithnature.org/how-do-cities-contribute-to-global-nature-goals-and-targets/' target='blank'>
+            CitiesWithNature Action Platform
+            </a>
```
Comment (yndajas) [inline config/locales/about/en.yml:25]: This line break makes the space after the text part of the anchor tag

File: app/views/about/show.html.erb [hunk 2]
```diff
@@ -2,17 +2,12 @@
...
-<div class="about__cards">
-  <div class="about__cards-card">
-    <%= I18n.t('about.footer.left') %>
-  </div>
-
-  <div class="about__cards-card">
-    <%= I18n.t('about.footer.right') %>
+<div class='about__cards'>
+  <div class='about__cards-card'>
+    <%= I18n.t('about.footer').html_safe %>
```
Comment (yndajas) [inline app/views/about/show.html.erb:11]: This change seems to have made the left and right sides divide a little less cleanly

Before the split was consistently at the end of a sentence:

Screenshot 2022-03-28 at 18 04 37

Now it's random based on screen width, and commonly mid-sentence:

Screenshot 2022-03-28 at 18 04 44

**Review by yndajas — CHANGES_REQUESTED** [review]
Couple of quick style clean ups here

File: config/locales/about/en.yml (hunk 1 above)
Comment (sergiomarrocoli) [reply config/locales/about/en.yml]: fixed

File: app/views/about/show.html.erb (hunk 2 above)
Comment (sergiomarrocoli) [reply app/views/about/show.html.erb]: moved back into two cards

**Review by yndajas — APPROVED** [review]

**Pull request merged**